<launch>
  <arg name="ns" default="/kulbabu"/>

  <group ns="$(arg ns)">

    <!-- Load Ultrasonic Array -->
    <node name="kulbabu_hardware_ultrasonic" pkg="kulbabu_hardware" type="kulbabu_hardware_ultrasonic"
          output="screen">

      <param name="topic_name" type="string" value="range"/>

      <param name="i2c_adapter" type="int" value="1"/>
      <param name="i2c_address" type="int" value="16"/>

      <param name="field_of_view" type="double" value="0.261799388"/>
      <param name="max_range" type="double" value="4.00"/>
      <param name="min_range" type="double" value="0.02"/>
      <param name="publish_frequency" type="double" value="10.0"/>

    </node>

    <!-- Load hardware interface -->
    <node name="kulbabu_hardware_interface" pkg="kulbabu_hardware" type="kulbabu_hardware_main"
          output="screen">

      <param name="cycle_time_error_threshold" type="double" value="10.00"/>
      <param name="loop_hz" type="double" value="10.0"/>

    </node>

  </group>

</launch>
